.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+Use the following template. Feel free to remove any unused sections.
+
+## FILE 1 TITLE 
+
+### Section A
+* Change 1
+> * Link to GitHub Issue(s) / Trello Card(s) if applicable.
+
+### Section B
+* Change 2
+> * Link to GitHub Issue(s) / Trello Card(s) if applicable.
+
+## FILE 2 TITLE
+
+### Section A
+* Change 3
+> * Link to GitHub Issue(s) / Trello Card(s) if applicable.
```

Comment:
Replace "Link to" with "Links to any applicable".
Remove "if applicable."
Replace "/" with "or".

> * Links to any applicable GitHub Issues or Trello Cards.